### ISSUE TITLE
Bad request when sorting/filtering on study hierarchy

### ISSUE THREAD

**Issue by sam-glendenning** [ral-facilities/datagateway#783]
**Description**:
When sorting and filtering on ISIS study hierarchy and then reloading the page (or changing the view) an error 400 occurs. This happens because the filters that are being applied to the studies are also being applied to the investigations linked to the currently viewed instrument. For example, navigating to study hierarchy and sorting by name will work just fine with the following request being triggered:
```
https://scigateway-preprod.esc.rl.ac.uk/datagateway-api/studyinvestigations?order="study.name asc","id asc"&skip=0&limit=10&where={"investigation.id":{"in":[16,28,43,50,66,71,76,81,97,110,127,134,155,224,234,239]}}&include="investigation","study"
```
but then reloading the page will trigger the following query to the API:
```
https://scigateway-preprod.esc.rl.ac.uk/datagateway-api/investigations?order="study.name asc","id asc"&where={"investigationInstruments.instrument.id":{"eq":1}}&distinct="id"
```
which is obviously invalid because investigations do not have the study property. 

The fix should be relatively straightforward: keep the currently applied filters, just don't apply them to the investigations query.

**Acceptance criteria**:
- [ ] Sorting and filtering on study hierarchy, followed by a page refresh or a view change should work just fine!

**Comment by louise-davies** [comment]
@sam-glendenning I refactored how the study views work and now they only fetch studies and don't prefetch investigations at all - can you confirm this bug is fixed on the react-query branch and if so we can say it will be closed with the PR

**Comment by sam-glendenning** [comment]
I'll check it tomorrow and close if required.

**Comment by sam-glendenning** [comment]
This issue is being resolved in https://github.com/ral-facilities/datagateway/pull/701/commits/56024b88f1a86d182d3f08388091a0f49b2a2f2b as part of https://github.com/ral-facilities/datagateway/pull/701. A single request is now made to the `studies` endpoint to link `investigations` and `studies`, compared to the previous solution which made requests to `investigations` and `studyInvestigations` endpoints.

@louise-davies I do notice one outstanding problem however. Sorting by study description was previously an option but this has been replaced by sorting by investigation title. The label at https://github.com/ral-facilities/datagateway/blob/6d6bdd04da7354ce409eca188454e942474ea1f5/packages/datagateway-dataview/src/views/card/isis/isisStudiesCardView.component.tsx#L89 is invalid because studies do not have a title field. Did you mean to keep the original sorting by description but used the wrong label or do you mean to allow for sorting by investigation title? Either way, the label needs changing

**Comment by louise-davies** [comment]
@sam-glendenning The switch from description to investigation title is due to #742 and now sorting may not work due to me not updating preprod to include the fix in ral-facilities/datagateway-api#420. The label is my bad - I just forgot to add it to `default.json`
